test/test.ts

```diff
@@ -7,11 +7,15 @@ import test, {ExecutionContext} from 'ava';
 import {JSDOM} from 'jsdom';
 import {Subject, Observable} from 'rxjs';
 import ZenObservable from 'zen-observable';
-import is, {TypeName} from '../source';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
```

Comment:
Either `import/default` follows `esModuleInterop` and therefore allows `import X from Y` on CJS modules, or `@typescript-eslint/no-require-imports` should be disabled.

Since I have to disable a rule, I might as well leave the old ESM import and disable `import/default` on that line